### DISCUSSION TITLE
Did not find dashboard.blade.php

### DISCUSSION THREAD

**Discussion by chandanpatra79** [Laravel-Backpack/community-forum#739] [Q&A (Help)] [answered]
Hi

Installation Guide suggest to copy "vendor/backpack/theme-coreuiv4/src/resources/views/dashboard.blade.php" to "/views/vendor/backpack/theme-coreuiv4/.." folder to override the Default Dashboard view. 

-- I did not see this file anywhere inside the code. Can you please suggest a way-out here ? 
-- I am trying to see the CoreUIv4 Dashboard with some Charts etc embedded ( as described on the website )

Thanks

**Comment by tabacitu** [comment, marked as answer]
Hey @chandanpatra79 ,

Take a look at the docs here: https://backpackforlaravel.com/docs/6.x/base-how-to#customize-the-dashboard - they provide both the steps AND the code for that view that you can copy-paste.

Note that you can put it inside `views/vendor/backpack/theme-coreuiv4`, yes, but you can also put it inside `views/vendor/backpack/ui` - both should work fine. I'd argue putting it in UI is better because if you change themes later on... it'll still be picked up.

Cheers!

**Reply by tabacitu** [reply]
PS. Where did you see those instructions?

**Reply by tabacitu** [reply]
You can also see how we do it in our demo here - https://github.com/Laravel-Backpack/demo/tree/main/resources/views/vendor/backpack

**Reply by chandanpatra79** [reply]
Thank you for your reply. I got what I was looking for, I found that Demo version has some more elements/examples.
